.github/release-drafter.yml

```diff
@@ -0,0 +1,38 @@
+# Config for https://github.com/release-drafter/release-drafter
+name-template: 'Akka Serverless Javascript/Typescript SDK $NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Productboard Features'
+    labels:
+      - 'Feature'
+  - title: ':toolbox: Chore'
+    labels:
+      - 'chore'
+  - title: 'Documentation'
+    labels:
+      - 'Documentation'
+  - title: 'TCK'
+    labels:
+      - 'tck'
+
+change-template: '- $TITLE #$NUMBER by @$AUTHOR'
+template: |
+  # :mega: Akka Serverless Java SDK $NEXT_PATCH_VERSION
+
+  ## Changes
+
+  $CHANGES
+
+  ## :bow: Credits
+  Special thanks to the following contributors who helped with this release: $CONTRIBUTORS
+
+autolabeler:
+  - label: 'chore'
+    title:
+      - '/chore:/i'
+  - label: 'Documentation'
+    files:
+      - 'docs/*'
+  - label: 'tck'
+    files:
+      - 'tck/*'
```

Comment:
Should we add other conventional commit types that we'll be using, maybe `fix` in particular for a fixes section?